### ISSUE TITLE
Option to play when only overflow

### ISSUE THREAD

**Issue by vadymfrlv** [justin-chu/react-fast-marquee#103]
I think it would be useful to have a property that would allow avoiding the use of marquee if there is no overflow.

**Comment by Torenrob** [comment]
I agree with this. I tried to hack this in my project but it was hard for me to figure out how to access the properties of the inner children and apply to the component.

**Comment by oldo** [comment]
You should be able to use a measure hook to measure the width of the content and the container and determine whether overflow has occurred.

https://usehooks.com/usemeasure

**Comment by Minding000** [comment]
Here's a version that supports `on-overflow` (enabled by default): https://www.npmjs.com/package/tiny-marquee
